cdc/kv/client.go

```diff
@@ -53,11 +53,13 @@ import (
 )
 
 const (
-	dialTimeout               = 10 * time.Second
-	maxRetry                  = 100
-	tikvRequestMaxBackoff     = 20000   // Maximum total sleep time(in ms)
-	grpcInitialWindowSize     = 1 << 27 // 128 MB The value for initial window size on a stream
-	grpcInitialConnWindowSize = 1 << 27 // 128 MB The value for initial window size on a connection
+	dialTimeout           = 10 * time.Second
+	maxRetry              = 100
+	tikvRequestMaxBackoff = 20000 // Maximum total sleep time(in ms)
+	// TODO find optimal values and test extensively before releasing
+	// The old values cause the gRPC stream to stall for some unknown reason.
+	grpcInitialWindowSize     = 1 << 28 // 256 MB The value for initial window size on a stream
+	grpcInitialConnWindowSize = 1 << 29 // 512 MB The value for initial window size on a connection
```

Comment:
I'm afraid gRPC may consume too much memory when there are multiple TiKV (N). Total memory = N * 512MB.

Reply:
I might be a better idea to sort this out in a future PR, because when I ran tests, a small gRPC window size is more likely to cause problems such as sudden decrease in throughput.